src/main/java/swaglabs/common/CommonMethods.java

```
@@ -101,5 +103,31 @@ public <T extends WebElement> void addToCartAndVerifyItemAdded(T addToCartElemen
 		logger.info("Element is added into Cart and Cart Value is " + cartCount);
 		waitForElementContainText(swagLabsHomePage.nonEmptyCartBadge, String.valueOf(cartCount + 1), 10);
 	}
+	
+	public static void loginPageButton(String buttonType) {
+		long l=3000000000l;
+		SwagLabsHomePage swagLabsHomePage = new SwagLabsHomePage(driver);
+		if(buttonType.equalsIgnoreCase("Login")) waitForElementContainText(swagLabsHomePage.logoHeading, "Swag Labs", 10);
+		try {

```

Empty catch block silently swallows exception. As per Google coding standards, this is incorrect.